bin/fuel-core/src/cli/run/p2p.rs

```diff
@@ -45,7 +45,7 @@ pub struct P2PArgs {
     pub public_address: Option<Multiaddr>,
 
     /// p2p network's TCP Port
-    #[clap(long = "peering_port", default_value = "4001", env)]
+    #[clap(long = "peering_port", default_value = "30333", env)]
```

Comment:
What's the main reason for changing the p2p port?

Reply:
Because it is the usual port for the p2p. + Our help chart uses [this port](https://github.com/FuelLabs/fuel-core/pull/954/files#diff-36f9a9124e36f26a09cbf2acb98d5cf34a5dbbe1d9d9a9931df4327deb22bd3b)